Only in tzu: intro
Only in lao: preamble
<fold text='diff -ur lao/quote tzu/quote' expand='true'>diff -ur lao/quote tzu/quote
--- lao/quote	2019-02-23 14:40:37.000000000 +0100
+++ tzu/quote	2019-02-23 14:45:39.000000000 +0100
<fold text='@@ -1,7 +1,6 @@' expand='true'>@@ -1,7 +1,6 @@
-The Way that can be told of is not the eternal Way;
-The name that can be named is not the eternal name.
 The Nameless is the origin of Heaven and Earth;
-The Named is the mother of all things.
+The named is the mother of all things.
+
 Therefore let there always be non-being,
   so we may see their subtlety,
 And let there always be being,</fold>
<fold text='@@ -9,3 +8,6 @@' expand='true'>@@ -9,3 +8,6 @@
 The two are the same,
 But after they are produced,
   they have different names.
+They both may be called deep and profound.
+Deeper and more profound,
+The door of all subtleties!
\ No newline at end of file</fold>
Only in tzu: unquote
</fold>